tuf_conformance/simulator_server.py

```diff
@@ -79,6 +81,10 @@ def new_test(self, name: str) -> tuple[ClientInitData, RepositorySimulator]:
             repo.fetch_metadata("root", 1),
         )
 
+        # The repository simulator publishes the root.
+        # Here, we publish the remaining metadata:
+        repo.publish([Targets.type, Snapshot.type, Timestamp.type])
```

Comment:
I would like this to happen in the same place where root is published or to have a a reason why it is hidden in the http server.

Reply:
https://github.com/theupdateframework/tuf-conformance/pull/188/commits/2caf72a241205ef262c5ee2fd58efcfb665b98c6